From 7e04d54de0ee83fd9c944962c2daa1cc745bbcb7 Mon Sep 17 00:00:00 2001
From: Sergey Poznyakoff <gray@Mirddin.farlep.net>
Date: Mon, 5 Jan 2009 16:12:39 +0000
Subject: [PATCH] More fixes.

* ping/ping_common.h (DEFAULT_PING_COUNT): Set to 0. The
standard behavior for ping is to keep sending packets until
the user cancels it (unless -c is given).

---
 ping/ping_common.h |    2 +-
 5 files changed, 10 insertions(+), 6 deletions(-)

diff --git a/ping/ping_common.h b/ping/ping_common.h
index a3618e4..2adeeea 100644
--- a/ping/ping_common.h
+++ b/ping/ping_common.h
@@ -50,7 +50,7 @@ struct ping_stat
 
 #define PING_CKTABSIZE 128
 
-#define DEFAULT_PING_COUNT 4
+#define DEFAULT_PING_COUNT 0
 
 #define PING_TIMING(s) (s >= PING_HEADER_LEN)
 #define PING_HEADER_LEN sizeof (struct timeval)
-- 
1.6.3.3

